### PR TITLE
Add microsoft-azure-storage-connector-example.md

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

**Comment by KasunAratthanage** [comment]
This PR content included in following PR.
https://github.com/wso2/docs-ei/pull/1400

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+
+If you do not want to build this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
+
+{!references/connectors/importing-connector-to-integration-studio.md!}
+
+1. Right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:43]: Integration Project

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+2. Specify the API name as `MSAzureStorage` and API context as `/resources`. You can go to the source view of the XML configuration file of the API and copy the following configuration (source view).
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:45]: You can go to the XML configuration of the API (source view) and copy the following configuration.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+               </log>
+               <respond/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+   ```
+**Note**
+
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:176]: Please modify below properties of the configuration  as applicable.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+               "employeefinancedetails"
+            ]
+         }
+      }`
+
+3. Upload text or binary employee data (blob).
+
+   **Sample request**
+   
+     `curl -v POST -d {"containerName": "employeedetails","fileName": "sample.txt","filePath": "/home/kasun/Documents/MSAZURESTORAGE/sample.txt"} "http://localhost:8290/resources/adddetails" -H "Content-Type:application/json"`
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:236]: Let's mention `/home/kasun/Documents/MSAZURESTORAGE/sample.txt` should be a valid path to a text file containing employee information.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md [hunk 5]
```diff
@@ -0,0 +1,286 @@
...
+     
+     `curl -v POST -d {"containerName": "employeedetails"} "http://localhost:8290/resources/listdetails" -H "Content-Type:application/json"`
+
+   **Expected Response**
+   
+     It will retrieve the uploaded text or binary name and the file path.
+     
+     `{
+          "result": {
+              "blob": "http://eiconnectortest.blob.core.windows.net/employeedetails/sample.txt"
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:256]: You can use this url to get the actual content of the file? If so, please mention it. You can even use WSO2 EI itself to read the file.

Comment (abeykoon) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:256]: Shall we implement this as well?

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:43]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:45]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:176]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:236]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-connector-example.md:256]: updated.

**Comment by KasunAratthanage** [comment]
Please find this PR content and suggested fixed comments are available in the following PR.
https://github.com/wso2/docs-ei/pull/1400

**Pull request closed without merging**